### PR TITLE
Lexus is adjustment

### PR DESCRIPTION
**Pull request by eFiniLan**
× Add 581 for ISH gas pedal signal
image

× Add STEER_ANGLE in 608 STEER_TORQUE_SENSOR (similar to TSS2)
image

× Change STEER_TORQUE_EPS factor in 608 STEER_TORQUE_SENSOR to 1.0

### CODE REVIEW

**Comment by quillford** [comment]
Just checked one of my routes, and it definitely seems like the factor needs to be increased. 0.77 seems to fit better though.

**Comment by rbiasini** [comment]
Ok, happy to merge 0.77. just remember to change it in interface.py as well if you drive with it. And flash panda :)

**Comment by eFiniLan** [comment]
@rbiasini awesome! updated! are you ok with the other 2 changes? :)
@quillford suggest you to try STEER_ANGLE in 608 STEER_TORQUE_SENSOR too, I found it turn way smoother in 0.6.4/0.6.5.

**Comment by quillford** [comment]
Cabana graphs weren't working for me so I threw together [this script](https://gist.githubusercontent.com/quillford/1a771a2e23fb06bbf2d8720abf71efa3/raw/3fdc4eedd345f01ce8848b055e290771c5e37bca/eps_factor.py). should help with other ports.

File: generator/toyota/lexus_is_2018_pt.dbc [hunk 1]
```diff
@@ -6,15 +6,19 @@ BO_ 550 BRAKE_MODULE: 8 XXX
...
  SG_ GAS_RELEASED : 3|1@0+ (1,0) [0|1] "" XXX
  SG_ GAS_PEDAL : 55|8@0+ (0.005,0) [0|1] "" XXX
 
 BO_ 608 STEER_TORQUE_SENSOR: 8 XXX
- SG_ STEER_TORQUE_EPS : 47|16@0- (0.66,0) [-20000|20000] "" XXX
+ SG_ STEER_TORQUE_EPS : 47|16@0- (0.77,0) [-20000|20000] "" XXX
  SG_ STEER_TORQUE_DRIVER : 15|16@0- (1,0) [-32768|32767] "" XXX
  SG_ STEER_OVERRIDE : 0|1@0+ (1,0) [0|1] "" XXX
  SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
+ SG_ STEER_ANGLE : 31|16@0- (0.056,0) [-500|500] "" XXX
```
Comment (rbiasini) [inline generator/toyota/lexus_is_2018_pt.dbc]: 0.573 is more correct (rad/100). I'll fix it in a separate commit

**Pull request merged**